src/main/java/author/AuthorList.java

```diff
@@ -36,4 +36,15 @@ public Author getAuthor(Author author) {
     public void addAuthor(Author author) {
         authorList.add(author);
     }
+
+    public int size() {
```

Comment:
Perhaps this could be changed to `getSize` to better differentiate between this method and the underlying `size` method of the list!

Reply:
I don't fully agree with you. `size()` does the same thing as the underlying method of the `ArrayList`. I think what we can do it have `AuthorList` extend `ArrayList<Author>`, like how I did it in my iP: 
![image](https://github.com/user-attachments/assets/e0656a3d-2fd2-4cf9-befc-a048e569a122)

Then `size()` will just be an overridden method.